package.json

```diff
@@ -7,7 +7,11 @@
   "author": "ESPHome maintainers",
   "license": "Apache-2.0",
   "scripts": {
-    "prepublishOnly": "script/build"
+    "prepublishOnly": "script/build",
+    "lint:prettier": "prettier \"src/**/*.{js,ts,json,css,md}\" --check",
+    "format:prettier": "prettier \"src/**/*.{js,ts,json,css,md}\" --write",
+    "lint:types": "tsc",
+    "lint:lit": "lit-analyzer \"src/**/*.ts\""
```

Comment:
I don't see much need for these commands?

Reply:
My idea was to mimic actions that we have in HA Frontend and maybe add them in pre-commit. Especially prettier.

Reply:
In an ideal world, we have a pre-commit, and then run pre-commit in check mode in CI. Pre-commit is Python based, not sure if there is a JS variant or that we require people to have Python? (not the end of the world)

But if we do pre-commit, would we still need the tasks here? And we shouldn't add tasks unless we have pre-commit in place

Reply:
yeah, let's mimic HA frontend 👍